src/partials/layout.handlebars

```diff
@@ -53,7 +53,7 @@
                         <li><a href="/documentation/introduction.html">The REST API basics</a></li>
                         <li><a href="/events-documentation/introduction.html">The Events API basics</a></li>
                         <li><a href="/php-client/introduction.html">The PHP API client</a></li>
-                        <li><a href="/apps/introduction.html">Apps (available soon)</a></li>
+                        <li><a href="/apps/introduction.html">Apps (available for partners)</a></li>
```

Comment:
We should keep `soon`, writen this way, it feel like it's a feature reserved for partners.

Reply:
Already available for partners?

Reply:
I would advise against introducing confusion that this feature was released for partners early and only.
We are in the menu, we don't have a lot of space. Being precise here is impossible.